lpa-submissions-e2e-tests/cypress/integration/appealsPlanningOfficersReport.feature

```diff
@@ -0,0 +1,97 @@
+Feature: uploadReportToReachDecision
+  As a Beta LPA Planning Officer
+  I want to provide the Planning Inspectorate with a copy of the Planning Officers Report
+  So that this can form part of the Inspectors decision
+
+Scenario: AC1 Access Upload the Planning Officers report question
+Given LPA Planning Officer is within the LPA Questionnaire task list
+When LPA Planning Officer selects the Upload the Planning Officers Report link
+Then LPA Planning Officer is presented with the ability to upload the Planning Officers report
+
+Scenario Outline: AC2 LPA Planning Officer successfully uploads file via upload button
+Given Upload the Planning Officers report is requested
+When a '<valid_file>' is successfully uploaded
+Then progress is made to the Task List
+And Upload the Planning Officers report is shown as completed
+Examples:
+|valid_file|
+|upload-file-valid.tiff|
+|upload-file-valid.tif|
+|upload-file-valid.png|
+|upload-file-valid.pdf|
+|upload-file-valid.jpg|
+|upload-file-valid.jpeg|
+|upload-file-valid.doc|
+|upload-file-valid.docx|
+
+Scenario Outline: AC3 LPA Planning Officer successfully uploads file via Drag and Drop
+Given Upload the Planning Officers report is requested
+When a '<valid_file>' is uploaded via drag and drop
+Then progress is made to the Task List
+And Upload the Planning Officers report is requested
```

Comment:
Should this be:  And upload the Planning Officer's report subsection is shown as completed @swdpcomputing @shilpabasu22

Reply:
Done

Reply:
Great